adr/query_pushdown.md

```diff
@@ -0,0 +1,47 @@
+Select queries of the form `{_id: <ObjectID>}` (value type is `ObjectID`).
+
+## Postgres
+
+If value type is not `ObjectID`, fallback to fetch entire table.
```

Comment:
That requires more explanation: what fallback, how we will do it, etc. What will high-level changes in the source code be needed?